README.md

```diff
@@ -24,8 +24,18 @@ Shoot me a message that you're doing it though, to prevent us from doing the sam
 
 As I'm currently refusing to pay Google $$$ to be able to upload this to the Chrome Web Store you will have to install this manually.
 
-Download the newest version here:  
```

Comment:
trailing spaces are semantic in markdown and signify to break the line after - has to stay in